drop-in/src/main/java/com/adyen/checkout/dropin/internal/ui/DropInActivity.kt

```diff
@@ -132,6 +132,7 @@ internal class DropInActivity :
 
         override fun onServiceDisconnected(className: ComponentName) {
             adyenLog(AdyenLogLevel.DEBUG) { "onServiceDisconnected" }
+            serviceBound = false
```

Comment:
Is this necessary here?

I see that we already set `serviceBound` to `false` in `DropInActivity:245`

Reply:
`onServiceDisconnected` is only triggered when the service is destroyed by external factors, like the system having not enough memory. In that case it would be nice for us to know if the service is still bound or not

Reply:
I see, makes sense.